### PR TITLE
README: describe -V and --config

### PR DESCRIPTION
**Pull request by kolyshkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,26 +19,47 @@ make
...
+under [dist/releases/](./dist/releases/).
+
+A specific configuration from a file can be specified using
+`--config path/to/config.toml` option.
+
 ### Scan an OpenShift release payload
 
 ```sh
- sudo ./check-payload scan payload \
+ sudo ./check-payload scan payload -V 4.11 \
```
Comment (mrunalp) [inline README.md:41]: Let's change the example to released image - quay.io/openshift-release-dev/ocp-release:4.11.44-x86_64

File: README.md [hunk 2]
```diff
@@ -19,26 +19,47 @@ make
...
+Here
+* `-V` specifies the configuration for a particular OpenShift version;
+* `--url` specifies a payload URL;
+* `--output-file` specifies a file to write the scan report to.
+
 ### Scan a container or operator image
 
 ```sh
-sudo ./check-payload scan operator \
+sudo ./check-payload scan operator -V 4.11 \
```
Comment (mrunalp) [inline README.md:54]: For operator, most likely will need to use operator specific config. Initially they should start with no exemptions.

Comment (mrunalp) [reply README.md:54]: As a nit we can use some other example below instead of ocp-priv

File: README.md (hunk 1 above)
Comment (kolyshkin) [reply README.md:41]: fixed

File: README.md (hunk 2 above)
Comment (kolyshkin) [reply README.md:54]: right

**Review by mrunalp — APPROVED** [review]

**Pull request merged**